design_docs/oscode/oscode_sundials.md

```diff
@@ -63,19 +72,32 @@ Explain the proposal as if it was already included in the project and you were t
 
 For implementation-oriented RFCs (e.g. for compiler internals), this section should focus on how compiler contributors should think about the change, and give examples of its concrete impact. For policy RFCs, this section should provide an example-driven introduction to the policy, and explain its impact in concrete terms.
 
+<!--- Should the above be deleted? --->
```

Comment:
Yes when we send it over we will delete that part. It's just there to explain what this section should be about